scripts/check-cross-platform-scripts.mjs

```
@@ -0,0 +1,37 @@
+    .map(({ replacement }) => ({ name, command, replacement })),
+)
+
+if (failures.length > 0) {
+  console.error('Found package scripts that rely on Unix-specific shell commands:')
+
+  for (const { name, command, replacement } of failures) {
+    console.error(`- ${name}: ${command}`)
+    console.error(`  Use ${replacement} for Windows-compatible scripts.`)
+  }
+
+  process.exit(1)
```

ESLint

[error] 'process' is not defined.

(no-undef)